azure-pipelines-full-tests.yml

```diff
@@ -13,144 +13,74 @@ pr:
       - '*'
 
 jobs:
-  - job: ExtractMetadata
-    displayName: Extract Metadata
+- job: AutomationTest
+  displayName: Automation Test (Profile Latest)
+  timeoutInMinutes: 120
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
+  steps:
+    - template: .azure-pipelines/templates/automation_test.yml
+      parameters:
+        pythonVersion: '$(python.version)'
+        profile: 'latest'
+        fullTest: true
 
-    condition: succeeded()
-    pool:
-      vmImage: 'ubuntu-16.04'
-    steps:
-      - task: Bash@3
-        displayName: 'Extract Version'
-        inputs:
-          targetType: 'filePath'
-          filePath: scripts/release/get_version.sh
-      - task: PublishPipelineArtifact@0
-        displayName: 'Publish Artifact: metadata'
-        inputs:
-          TargetPath: $(Build.ArtifactStagingDirectory)
-          ArtifactName: metadata
+- job: AutomationTest20200901
+  displayName: Automation Test (Profile 2020-09-01)
+  timeoutInMinutes: 120
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
```

Comment:
Where is 3.7?

Reply:
We package with Python 3.6 and currently we support up to 3.8. @fengzhou-msft